src/features/profile/pages/LegalNotices/LegalNotices.tsx

```diff
@@ -6,12 +6,39 @@ import { env } from 'libs/environment'
 import { SectionRow } from 'ui/components/SectionRow'
 import { Separator } from 'ui/components/Separator'
 import { ExternalSite } from 'ui/svg/icons/ExternalSite'
-import { getSpacing } from 'ui/theme'
-import { SECTION_ROW_ICON_SIZE } from 'ui/theme/constants'
+import { getSpacing, Spacer, Typo } from 'ui/theme'
+import { LINE_BREAK, SECTION_ROW_ICON_SIZE } from 'ui/theme/constants'
 
 export function LegalNotices() {
   return (
-    <PageProfileSection title="Mentions légales">
+    <PageProfileSection title="Informations légales">
```

Comment:
Il faut ajouter `scrollable`, sinon on peut pas scroller sur les petits écrans (sur iPhone 5S par exemple, c'est coupé après le premier lien) : 
```suggestion
    <PageProfileSection title="Informations légales" scrollable>
```
<img src="https://user-images.githubusercontent.com/46637324/205086310-ab3a27fd-3dab-4010-90ad-54d5d5567d80.png" width="300" />